ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/PostgresLedgerDao.scala

```diff
@@ -992,7 +995,7 @@ private class PostgresLedgerDao(
   override def storeParty(
       party: Party,
       displayName: Option[String]): Future[PersistenceResponse] = {
-    dbDispatcher.executeSql { implicit conn =>
+    dbDispatcher.executeSql(s"store party [$party]") { implicit conn =>
```

Comment:
Could be a candidate for unwrapping. Not sure though, I can't see types properly in GitHub. :slightly_smiling_face:

Reply:
No need here, as `Party` eventually is a `LedgerString`.